source/reference/mongosync-behavior.txt

```diff
@@ -21,8 +19,13 @@ The ``mongosync`` binary is the primary process used in
 {+c2c-product-name+}. ``mongosync`` migrates data from one cluster to
 another and can keep the clusters in continuous sync. 
 
-This document provides a complete overview of the ``mongosync`` command
-line options.
+This document provides a complete overview of ``mongosync`` behavior. For
+information on how the ``mongosync`` process works, see :ref:`about-mongosync`. 
+
+To get started with ``mongosync``, refer to the :ref:`Quick Start Guide
+<c2c-quickstart>`. For more detailed information, refer to the
+:ref:`c2c-install` or :ref:`c2c-connecting` page that best fits your
+situation. 
 
 .. note::
```

Comment:
[question] Do you think it makes sense to keep this note here or is there another place/page that this might fit better?

Reply:
Yes, I do think it makes sense for this note to be here. I think it summarizes important facts that are scattered in different places, and I think that users will look to the behavior page for this type of information. There is the limitations page, but we link to it in this note and I think it's important for these limitations to be at the top of the behavior page. What are your thoughts?

Reply:
That sounds like a fair reason to me!